src/components/settings/SettingsAccessTab/index.jsx

```diff
@@ -34,6 +42,10 @@ const SettingsAccessTab = ({
 
   return (
     <div className="settings-access-tab mt-4">
+      {subsidyRequestConfiguration?.subsidyType === SUPPORTED_SUBSIDY_TYPES.coupon
```

Comment:
I think we also want to check that requests are enabled.

Reply:
👍